gateway/ln-gateway/src/bin/cln_extension.rs

```diff
@@ -476,33 +475,26 @@ impl GatewayLightning for ClnRpcService {
 
     async fn route_htlcs(
         &self,
-        request: tonic::Request<tonic::Streaming<InterceptHtlcResponse>>,
+        _: tonic::Request<EmptyRequest>,
     ) -> Result<tonic::Response<Self::RouteHtlcsStream>, Status> {
-        let mut stream = request.into_inner();
-
         // First create new channel that we will use to send responses back to gatewayd
         let (gatewayd_sender, gatewayd_receiver) =
             mpsc::channel::<Result<InterceptHtlcRequest, Status>>(100);
 
         let mut sender = self.interceptor.sender.lock().await;
         *sender = Some(gatewayd_sender.clone());
 
-        // Spawn new thread that listens for events from the input stream
-        let interceptors = self.interceptor.clone();
```

Comment:
I'm super confused why you can just delete it here, and leave this function as is. :D

Reply:
This code was processing a stream of `InterceptHtlcResponse` from gatewayd. This PR changes the way gatewayd sends responses back, it now uses a separate RPC (`complete_htlc`). So this code is not necessary anymore.